tough/src/sign.rs

```diff
@@ -106,6 +106,7 @@ impl Sign for EcdsaKeyPair {
 
 /// Keypair used for signing metadata
 #[derive(Debug)]
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
Please let me know if you like this approach or if instead it would be better to change the variants of this enum to do not use uppercase acronyms (I guess that we would need to add an entry in the Changelog in that case)